doc/api/fs.md

```diff
@@ -1143,6 +1143,9 @@ changes:
 Synchronously changes the permissions of a file. Returns `undefined`.
 This is the synchronous version of [`fs.chmod()`][].
```

Comment:
@iwko I believe @davisjam and @Trott and I meant that sentence like 

```
 This is the synchronous version of [`fs.chmod()`][].
```

Should be removed after sentences like

```
For detailed information, see the documentation of the asynchronous version of
this API: [`fs.chmod()`][].
```

are added to avoid duplication?

Reply:
I am a little confused now. So documentation for `fs.chmod()` should look like this?

`
Synchronously changes the permissions of a file. Returns `undefined`.

For detailed information, see the documentation of the asynchronous version of
this API: [`fs.chmod()`][].

See also: chmod(2).
`?

Reply:
@iwko Yes, that's my understanding

Reply:
@joyeecheung done